proposals/lip-0041.md

```diff
@@ -446,11 +449,8 @@ This function verifies the nonce of a given transaction `trs` and returns a bool
 
 ```python
 def isValidNonce(trs: Transaction) -> bool:
-    try:
-        verifyNonce(trs)
-        return True
-    except:
-        return False
+    nonceStatus = verifyNonce(trs)
+    return nonceStatus == NONCE_VERIFY_STATUS_OK
```

Comment:
It could be one line and the additional variable `nonceStatus` seems unnecessary.